docs/source/common/optimizers.rst

```diff
@@ -230,6 +230,21 @@ If you want to call ``lr_scheduler.step()`` every ``n`` steps/epochs, do the fol
         if self.trainer.is_last_batch and (self.trainer.current_epoch + 1) % n == 0:
             sch.step()
 
+If you want to call schedulers that require a metric value after each epoch, consider doing the following:
+
+.. testcode::
+
+    def __init__(self):
```

Comment:
can we add class too?

Reply:
I don't think it's required.

Reply:
Following the consistency with other demo codes in the doc.